Assets/VRTK/Scripts/Interactions/Highlighters/VRTK_OutlineObjectCopyHighlighter.cs

```diff
@@ -145,10 +145,10 @@ private void CreateHighlightModel()
             }
 
             highlightModel = new GameObject(name + "_HighlightModel");
-            highlightModel.transform.SetParent(transform);
-            highlightModel.transform.position = copyModel.transform.position;
-            highlightModel.transform.rotation = copyModel.transform.rotation;
-            highlightModel.transform.localScale = Vector3.one;
+            highlightModel.transform.SetParent(copyModel.transform.parent, false);
```

Comment:
This line has actually broken the Outline Highlighter. If you set it as the parent then the outline no longer follows the object around.